config/buildEnv.cmake

```diff
@@ -153,142 +173,155 @@ macro( dbsSetDefaults )
-    message(FATAL_ERROR
-      "The Draco build system doesn't currently allow the generation of object-libraries with"
-      "Visual Studio projects.")
+  if(DBS_GENERATE_OBJECT_LIBRARIES AND "${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+    message(
+      FATAL_ERROR
+        "The Draco build system doesn't currently allow the generation of object-libraries with"
+        "Visual Studio projects.")
   endif()
 
   # Enable parallel build for Eclipse:
-  cmake_host_system_information( RESULT logical_cores QUERY
-    NUMBER_OF_LOGICAL_CORES )
-  set( CMAKE_ECLIPSE_MAKE_ARGUMENTS "-j ${logical_cores}" )
+  cmake_host_system_information(RESULT logical_cores QUERY NUMBER_OF_LOGICAL_CORES)
+  set(CMAKE_ECLIPSE_MAKE_ARGUMENTS "-j ${logical_cores}")
 
   # Set RPATH for all libraries on Apple platform
-  if (APPLE)
+  if(APPLE)
     set(CMAKE_MACOSX_RPATH 1)
   endif()
 
-  if( "${DRACO_LIBRARY_TYPE}" MATCHES "SHARED" )
-     # Set replacement RPATH for installed libraries and executables. See
-     # http://www.cmake.org/Wiki/CMake_RPATH_handling
-
-     # Do not skip the full RPATH for the build tree
-     set( CMAKE_SKIP_BUILD_RPATH OFF )
-     # When building, don't use the install RPATH already (but later on when
-     # installing)
-     set( CMAKE_BUILD_WITH_INSTALL_RPATH OFF )
-
-     # For libraries created within the build tree, replace the RPATH in the
-     # installed files with the install location.
-     set( CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib" CACHE PATH
-       "RPATH to embed in dynamic libraries and executables when
-targets are installed." FORCE )
-
-     # add the automatically determined parts of the RPATH which point to
-     # directories outside the build tree to the install RPATH
-     set( CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE )
+  if("${DRACO_LIBRARY_TYPE}" MATCHES "SHARED")
+    # Set replacement RPATH for installed libraries and executables. See
+    # http://www.cmake.org/Wiki/CMake_RPATH_handling
+
+    # Do not skip the full RPATH for the build tree
+    set(CMAKE_SKIP_BUILD_RPATH OFF)
+    # When building, don't use the install RPATH already (but later on when installing)
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH OFF)
+
+    # For libraries created within the build tree, replace the RPATH in the installed files with the
+    # install location.
+    set(CMAKE_INSTALL_RPATH
+        "${CMAKE_INSTALL_PREFIX}/lib"
+        CACHE PATH "RPATH to embed in dynamic libraries and executables when
+targets are installed." FORCE)
+
+    # add the automatically determined parts of the RPATH which point to directories outside the
+    # build tree to the install RPATH
+    set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+  endif()
+
+  # -------------------------------------------------------------------------------------------- #
+  # Special FMA modes - roundoff controls
+  # -------------------------------------------------------------------------------------------- #
+  if(NOT DEFINED DRACO_ROUNDOFF_MODE)
```

Comment:
@alexrlongne This block is new in `buildEnv.cmake`.  It was more-or-less copied from Jayenne.